WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift

```diff
@@ -168,6 +168,10 @@ public struct WordPressAuthenticatorConfiguration {
     ///
     let enableSiteAddressLoginOnlyInPrologue: Bool
 
+    /// If enabled, the email login screen should not hide the site credential login CTA.
+    ///
```

Comment:
Super nit: Would it help to mention the default value as a comment?

Reply:
Sure - updated in 27b3eb0e0eeb24329508b17b366d3a80140f3a6c.